### PR TITLE
Updating spacy notebook

### PR DESCRIPTION
**Pull request by nadinet**
**What this PR does / why we need it**: 
Updating spacy example notebook

**Which issue(s) this PR fixes**:

Part of Issue #3072

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **gsunner**
You can assign the PR to them by writing `/assign @gsunner` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: examples/models/sklearn_spacy_text/Dockerfile
Comment (adriangonz) [inline examples/models/sklearn_spacy_text/Dockerfile:3]: Good one on pinning the dependency version!

File: examples/models/sklearn_spacy_text/reddit_clf.json
Comment (adriangonz) [inline examples/models/sklearn_spacy_text/reddit_clf.json:48]: If you've got time, it would be good to remove these extra spaces.

**Review by adriangonz — COMMENTED** [review]
Nice one! I just added a few comments below.

File: examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb
Comment (adriangonz) [inline examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb:11]: <p>It would be good to remove the outputs before merging the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Review by adriangonz — CHANGES_REQUESTED** [review]
Just as an example of how it would look like if I requested changes.

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3076#issuecomment-802942172 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by axsaucedo — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3076#pullrequestreview-616564378 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
